cpp/daal/src/data_management/roc_auc_score.cpp

```diff
@@ -113,16 +113,27 @@ services::Status rocAucScoreImpl(const NumericTablePtr & truePrediction, const N
     return s;
 }
 
+template <typename DataType>
 DAAL_EXPORT double rocAucScore(const NumericTablePtr & truePrediction, const NumericTablePtr & testPrediction)
 {
     double score = double(0);
-#define DAAL_ROC_AUC_SCORE(cpuId, ...) rocAucScoreImpl<cpuId>(__VA_ARGS__);
+#define DAAL_ROC_AUC_SCORE(cpuId, ...) rocAucScoreImpl<DataType, cpuId>(__VA_ARGS__);
 
     DAAL_DISPATCH_FUNCTION_BY_CPU_SAFE(DAAL_ROC_AUC_SCORE, truePrediction, testPrediction, score);
 
 #undef DAAL_ROC_AUC_SCORE
     return score;
 }
+
+template DAAL_EXPORT double rocAucScore<float>(const NumericTablePtr & truePrediction, const NumericTablePtr & testPrediction);
+template DAAL_EXPORT double rocAucScore<double>(const NumericTablePtr & truePrediction, const NumericTablePtr & testPrediction);
+
+// necessary for maintaining ABI
```

Comment:
Should we mark this so it would be easy to  identify and remove in 2026.x release?